src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JsonFeignResponseUtilTest.java

```diff
@@ -55,12 +52,12 @@ void test_Decode_fails_with_ioException() {
         Response response = Response.builder().status(200).reason("OK").headers(header).body(bodyMock)
                 .request(mock(Request.class)).build();
 
-        try {
-            when(bodyMock.asInputStream()).thenThrow(new IOException());
-            when(bodyMock.asReader(Charset.defaultCharset())).thenThrow(new IOException());
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
+        //      try {
+        //          when(bodyMock.asInputStream()).thenThrow(new IOException());
+        //          when(bodyMock.asReader(Charset.defaultCharset())).thenThrow(new IOException());
+        //      } catch (IOException e) {
+        //          e.printStackTrace();
+        //      }
```

Comment:
Why is this being commented out?

Reply:
Checked with team, Its unused code so removed